opa-manager-operator/metadata.yaml

```diff
@@ -0,0 +1,6 @@
+name: gatekeeper-manager
+description: |
+  This charm deploys OPA.
+summary: |
+  Kubernetes OPA operator
```

Comment:
Description and summary need to be updated. This charm deploys the Gatekeeper Controller Manager.

Reply:
updated